.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: required
```

Comment:
OK. You may fork [my PR](https://github.com/szepeviktor/realpath_turbo/tree/patch-1).
I suggest to call the ini `realpath_turbo.ini`